### PR TITLE
Security wednesdays patch

### LINKED ISSUE #7: Update Security Wednesdays
- [x] Add Webpage for Security Wednesdays
- [ ] Add link for participants
- [ ] Add link for speakers who want to contribute

### PR DESCRIPTION
**Pull request by Jh123x**
Closes #7 

- [x] Add Security Wednesday Page

In another PR
- [ ] Add Link to sign up as speaker
- [ ] Add Link to sign up as participant

### CODE REVIEW

File: security_wednesdays.html [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+
+					filter: selector,
+				});
+				return false;
+			});
+
+		});
+	</script>
+	
+	<script>
```
Comment (daniellimws) [inline security_wednesdays.html:574]: Er any clue where this google analytics links to?

Comment (Jh123x) [reply security_wednesdays.html:574]: No clue. I can remove it if it is not needed

Comment (daniellimws) [reply security_wednesdays.html]: I suppose we can change it to our own google analytics when we set that up

**Comment by daniellimws** [comment]
The fonts arent loaded properly it seems

**Comment by daniellimws** [comment]
For those with same date, e.g. this 

image

Can make it more obvious that they are on the same date? Maybe removing the mic icon on the second talk may help?

**Comment by Jh123x** [comment]
> For those with same date, e.g. this
> 
> image
> 
> Can make it more obvious that they are on the same date? Maybe removing the mic icon and the date on the second talk may help?

Ookie done

**Comment by daniellimws** [comment]
Fixed the font, and made more changes in #13

**Comment by Jh123x** [comment]
Close as there is a duplicated PR with more updated info

**Pull request closed without merging**